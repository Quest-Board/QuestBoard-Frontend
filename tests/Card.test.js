import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../src/components/Card';

test('test1', () => {
    const component = renderer.create(
        <Card></Card>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});