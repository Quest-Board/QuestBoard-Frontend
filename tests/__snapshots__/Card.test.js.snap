// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot test 1`] = `
<div>
  Hello World
</div>
`;
